Code/GraphMol/MolEnumerator/MolEnumerator.h

```diff
@@ -237,6 +240,55 @@ class RDKIT_MOLENUMERATOR_EXPORT RepeatUnitOp : public MolEnumeratorOp {
   void initFromMol();
 };
 
+using stereo_flipper_t = std::shared_ptr<StereoFlipper>;
+
+struct RDKIT_MOLENUMERATOR_EXPORT StereoEnumerationOptions {
```

Comment:
Please document the options here.
You can copy the docs from the python file

Reply:
fixed! I mostly copied the python docs as is, but it should be noted the current functionality is incomplete!